docs/plans/2026-05-15-pkm-scribe-and-vault-template-plan.md

```
@@ -0,0 +1,827 @@
+Replace the existing `## Active Projects` block and insert two new sections, exactly as done in borgr. Final section order: `## Purpose`, `## Standards`, `## Active Projects`, `## Backlog Micro-items`, `## Someday Projects`, `## Recurring Responsibilities`, `## Resources`, `## Links`.
+
+- [ ] **Step 1: Confirm the anchor**
+
+Run: `grep -n "^## " "templates/hoards/obsidian-vault/60_Metadata/Templates/Area Note.md"`
+Expected: `## Purpose`, `## Standards`, `## Active Projects`, `## Recurring Responsibilities`, `## Resources`, `## Links`.
+
+- [ ] **Step 2: Replace the `## Active Projects` block**
+
+Find the block that runs from the `## Active Projects` heading through the closing ` ``` ` of its `dataview` block. The current block is:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)